src/main/java/oshi/hardware/platform/mac/MacGlobalMemory.java

```diff
@@ -86,6 +87,18 @@ public long getTotal() {
     }
 
     @Override
+    public long getSwapTotal() {
+        // TODO: for now, it returns 0

```

Comment:
On Mac, use a `sysctl()` call with the name `vm.swapusage`.
It will return a string like this:
total = 1024.00M  used = 440.75M  free = 583.25M  (encrypted)

Note that unlike Linux, total can increase (up to free space on drive) if more swap is needed.